libs/wingsdk/src/target-tf-gcp/function.ts

```diff
@@ -20,6 +21,11 @@ const FUNCTION_NAME_OPTS: NameOptions = {
   case: CaseConventions.LOWERCASE,
 };
 
+enum StorageBucketPermissions {
```

Comment:
The same permissions are defined [here](https://github.com/winglang/wing/pull/4376/files#diff-efe32a53e4d5756b6117f10699793545c18a00d1352c26bcc576b34884a60464R36), can we merge them to avoid duplicity?

Reply:
I'm moving this to shared-gcp/bucket-inflight.ts and then importing it from there to bucket.ts and function.ts preflight resource file to avoid duplicity. Thanks!!
```js
import { StorageBucketPermissions } from "../shared-gcp/bucket.inflight";
```